### DISCUSSION TITLE
contributions

### DISCUSSION THREAD

**Discussion by mmadoo** [hubblo-org/scaphandre#229] [Q&A] [answered]
On which branch (dev or main) should be done pull requests (the file https://github.com/hubblo-org/scaphandre/blob/dev/docs_src/contributing.md does not precise it) ?

**Comment by mmadoo** [comment, marked as answer]
Finaly I found the response in https://nvie.com/posts/a-successful-git-branching-model/#feature-branches

**Comment by bpetit** [comment]
Hi !

You're right, it is dev, and should be documented, thanks for raising this issue :)
